backend/src/routes/studentLog.ts

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+import { StudentLog } from '../database/models/studentLog';
+
+export const logRoutes = express.Router();
+
+logRoutes.get('/logs/:sessionId', (req, res) => {
```

Comment:
```suggestion
logRoutes.get('/:sessionId', (req, res) => {
```
myślę że tutaj wystarczyłaby ścieżka ```'/:sessionId'``` - zauważmy że w index.ts mamy ```app.use('/studentLog', logRoutes);``` - co w aktualnej sytuacji daje ```'.../studentLog/logs/:sessionId'```